lib/src/main/kotlin/com/lemonappdev/konsist/core/ext/KoBaseDeclarationSequenceExt.kt

```diff
@@ -0,0 +1,15 @@
+import com.lemonappdev.konsist.core.declaration.KoBaseDeclaration
+
+fun <T : KoBaseDeclaration> Sequence<T>.withParentDeclaration(vararg names: String) = filter {
```

Comment:
this is very specific example, but if we want to keep it (for API completion) then we should accept  koDeclaration instance, not a name